src/resolve/resolved-manager.h

```diff
@@ -32,6 +32,13 @@ typedef struct EtcHosts {
         Set *no_address;
 } EtcHosts;
 
+/* A single node in a linked list of connections */
+typedef struct VarlinkConnection VarlinkConnection;
+struct VarlinkConnection {
+        Varlink* link;
+        VarlinkConnection* next;
```

Comment:
no manual linked lists please.

use the macros from list.h. Or even better just put the varlink clients to notify in a `Set` object.

Reply:
I've replaced this with a Set, it is much cleaner, thanks for the suggestion!